R/tax_range_time.R

```diff
@@ -157,7 +173,12 @@ tax_range_time <- function(occdf,
     if (by == "LAD") {
       temp_df <- temp_df[order(temp_df$min_ma), ]
       temp_df$taxon_id <- seq_len(nrow(temp_df))
-      }
+    }
+
+    if (is.null(group)) {
```

Comment:
```suggestion
    if (!is.null(group)) {
```
Is the "!" missing here?

Reply:
Ah, that would explain why the `group` variable wasn't changing the plot for me.